cve_bin_tool/checkers/libevent.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for bash
+https://www.cvedetails.com/product/32303/Libevent-Project-Libevent.html?vendor_id=15590
+"""
+from . import Checker
+
+
+class LibeventChecker(Checker):
+    CONTAINS_PATTERNS = []

```

Comment:
You haven't added anything inside CONTAINS_PATTERNS here.
Use the strings command and find some strings which look unique to this binary

Reply:
Thank you for your suggestion I have updated it 👍